hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonMultiBlockSplit.java

```diff
@@ -44,17 +44,17 @@
   /*
    * The location of all wrapped splits belong to the same node
    */
-  private String location;
+  private String[] locations;
```

Comment:
In case of bucketing we cannot maintain the multiblock split to single location, because the return splits should always be dependent on bucket splits.
I have modified the comment